### DISCUSSION TITLE
example_simple_pyspark.html not found

### DISCUSSION THREAD

**Discussion by theimanph** [moj-analytical-services/splink#1997] [General]
Hi,

I am running into errors trying to use pyspark. The error says to go to:https://moj-analytical-services.github.io/splink/demos/example_simple_pyspark.html to find out how to register the Splink jar. I get a 404 - Not found when I tried to look at it.. Is that location down? Thank you!!

tom

**Comment by RobinL** [comment]
Sorry, must be an issue with our docs build.  Correct link is https://moj-analytical-services.github.io/splink/demos/examples/spark/deduplicate_1k_synthetic.html

**Reply by theimanph** [reply]
Hi Robin, Thank you!! Where is the jar file located? I dont see it in the repo?

**Reply by theimanph** [reply]
Hi Robin, I went to https://github.com/moj-analytical-services/splink/blob/master/jars/scala-udf-similarity-0.0.7.jar for the jar and it doesnt appear to be there..

**Reply by RobinL** [reply]
https://github.com/moj-analytical-services/splink/tree/master/splink/files/spark_jars

**Reply by theimanph** [reply]
Hi Robin, Thank you!!!!
